src/index.js

```diff
@@ -103,7 +108,13 @@ funbtn.forEach((ele) => {
 // remove all
 const clear = document.querySelector('.clearAll');
 clear.addEventListener('click', () => {
-  doLists = [];
-  localStorage.setItem('Do-Lists', JSON.stringify(doLists));
-  window.location.reload();
+  clearChecked(doLists);
+});
+
+const check = document.querySelectorAll('.box');
+
+check.forEach((ele) => {
+  ele.addEventListener('click', () => {
+    checked(ele, doLists);
+  });
```

Comment:
- [ ] You can use the `change` event instead of `click` to ensure you are getting the true state of the checkbox so that when unchecked it can also update the localStorage